CHANGES.rst

```diff
@@ -638,15 +638,6 @@ astropy.table
 - Added optional ``axis`` parameter to ``insert`` method for ``Column`` and
   ``MaskedColumn`` classes. [#6092]
 
-- The private ``_parent`` attribute in the ``info`` attribute of table
-  columns was changed from a direct reference to the parent column to a weak
-  reference.  This was in response to a memory leak caused by having a
-  circular reference cycle.  This change means that expressions like
-  ``col[3:5].info`` will now fail because at the point of the ``info``
-  property being evaluated the ``col[3:5]`` weak reference is dead.  Instead
-  force a reference with ``c = col[3:5]`` followed by
```

Comment:
Just a note that this behavior wasn't ideal anyway (``col[3:5].info`` not working) so maybe we can find a better solution in future.